src/Command/Vendors/VendorLoadCommand.php

```diff
@@ -59,9 +61,28 @@ protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $limit = (int) $input->getOption('limit');
         $dispatchToQueue = !$input->getOption('without-queue');
-        $withUpdates = (bool) $input->getOption('with-updates');
         $force = (bool) $input->getOption('force');
 
+        $date = $input->getOption('with-updates-date');
+        $withUpdatesDate = \DateTime::createFromFormat('Y-m-d', $date);
+        if (false === $withUpdatesDate) {
+            $output->writeln('<error>Unknown date format in --with-updates</error>');
+
+            return 1;
+        }
+
+        $daysAgo = $input->getOption('days-ago');
+        if (!empty($daysAgo)) {
+            $withUpdatesDate = new \DateTime();
```

Comment:
I don't like that we silently overwrite another input. Is sure to cause confusion at some point even if the help text for the command states that this will happen.

We should check if both options are given and throw an error if they are, stating that only one can be used at a time.

Reply:
Fixed